herbert_core/admin/@opt_config_manager/private/find_comp_type_.m

```diff
@@ -50,14 +44,16 @@
         pc_type = types{3};
         return;
     end
-    
+
     rez=strfind(mess,'NUMA node');
     % if lscpu returns more then one numa node strigs, first string defines
     % the number of numa nodes and all subsequent strings describe each
     % node. So, if there are more then 2 string, its more then one numa
     % node and we consider this computer to be an hpc system.
-    if numel(rez)>2; hpc_computer = true;
-    else;          hpc_computer = false;
+    if numel(rez)>2 || nproc>4
+        hpc_computer = true;
+    else
+        hpc_computer = false;
```

Comment:
```suggestion
    hpc_computer = numel(rez) > 2 || nproc > 4;
```

Reply:
very "matter of taste" suggestion. I would not fining this more readable then the previous form and of course unlike C especially  in a loop -- no difference for performance